src/routes/token/verify.ts

```diff
@@ -0,0 +1,24 @@
+import { sendError } from '@/errors';
+import { getClaims } from '@/utils';
+import { Joi, token } from '@/validation';
+import { RequestHandler } from 'express';
+import { ReasonPhrases } from 'http-status-codes';
+
+export const verifyTokenSchema = Joi.object({
+  token,
+}).meta({ className: 'VerifyTokenSchema' });
+
+export const verifyTokenHandler: RequestHandler<
+  {},
+  {},
+  { token?: string }
+> = async (req, res) => {
+  const authorization = req.body.token || req.headers.authorization;
+
+  try {
+    getClaims(authorization);
+    return res.send(ReasonPhrases.OK);
```

Comment:
I can't remember seeing this pattern elsewhere (`ResponsePhrases.OK`). Is this consistent with what we're doing elsewhere?

Reply:
Yes, of course! I personally don't like it, but can be improved/changed in a different PR

<img width="371" alt="image" src="https://user-images.githubusercontent.com/2267270/167792744-24956674-c13c-4df5-a3aa-145bd03d698f.png">

Reply:
Perfect! But, light mode?? 😅

Reply:
I knew someone will make a note of that! 🤣 It's synced with macOS system appearance